.trunk/trunk.yaml

```diff
@@ -12,11 +12,13 @@ plugins:
       ref: v0.0.17
       uri: https://github.com/trunk-io/plugins
     - id: configs
-      ref: v0.0.2
+      ref: v0.0.1
```

Comment:
why are we rolling this back?

Reply:
There's a bug in configs - @TylerJang27 can speak more to it

Reply:
got it, go ahead and merge then